changelog/20034.txt

```diff
@@ -0,0 +1,3 @@
+```release-note: bug
+secrets/aws: Revert changes that removed the lease on STS credentials, while leaving the new ttl field in place. [[GH-20034](https://github.com/hashicorp/vault/pull/20034)]
```

Comment:
I don't think the link is needed. The CL generation will take care of that.

```suggestion
secrets/aws: Revert changes that removed the lease on STS credentials, while leaving the new ttl field in place.
```

Reply:
Thanks!